example/nextjs/pages/login.js

```diff
@@ -26,7 +26,7 @@ const Login = ({profile, origin}) => {
   const redirect = encodeURIComponent(origin + '/')
   return (
     <div className='section'>
-      { !profile && ['Twitter', 'Google', 'GitHub', 'Reddit', 'Facebook', 'LinkedIn', 'Instagram'].map(service => (
+      { !profile && ['Twitter', 'Google', 'GitHub', 'Reddit', 'Facebook', 'LinkedIn', 'Instagram', 'Eventbrite'].map(service => (
```

Comment:
I will not feature every login strategy on the demo page. Could you remove this?